test-partner/Dockerfile.debug-partner

```diff
@@ -2,7 +2,7 @@ FROM registry.access.redhat.com/ubi8/ubi:latest
 
 RUN \
     yum -y update && \
-    yum -y install pciutils util-linux net-tools procps-ng && \
+    yum -y install pciutils util-linux net-tools procps-ng iputils iproute&& \
```

Comment:
iputils iproute&& -> iputils iproute &&
also, we don't need this file at all ? since we're removing the partner pod

Reply:
fixed. This file is needed to create the debug pod image